### PR TITLE
Front page

### PR DESCRIPTION
**Pull request by fdezcaminero**
First pull

### CODE REVIEW

**Review by Forison — COMMENTED** [review]
# Pull request under review by Addo

**Review by Forison — CHANGES_REQUESTED** [review]
# Hi @Caminero👏
You have done a good job so far🙂
Here a few fixes we have to make to get this project approved🙂
- [ ]  It should be made of nine different buttons with the Apple logo on the leftmost section and a search field on the rightmost section🙂
Let us add text to those links instead of using background images for each link.
Let us work on the search field and make it better [refer](https://web.archive.org/web/20140301004610/http://www.apple.com/)
- [ ] let us properly implement the use of semantic HTML and also take care of the horizontal scroll.🙂
here we can wrap the main part thus ```aside``` and the ```img``` can be wrapped in ```main```
Implement a background image on the ```main```  and give it a width of 100% .also ```.lil-links"``` should be 100% instead of 105% to eliminate the horizontal scroll.
here is how the page looks at my end
Screenshot from 2019-11-05 17-37-08

## Kindly make these changes and submit for review🙂

feel free to contact me on [LinkedIn](https://www.linkedin.com/in/forison-addo-558201159/), [Twitter](https://twitter.com/ADDOBOAKYE1) or slack for further clarification if necessary. I would be happy to assist you.
#### Happy coding🙂

**Comment by ebukaume** [comment]
## Changes Required! :stop_sign: 

Hi @fdezcaminero :wave: 

You are doing well however, there are some things you need to fix.

Remember that this project is about **building with backgrounds and gradients**. The _hero_ image is supposed to be set as background (_as CSS background-image or background -> url_)

After setting the background of the hero image, those 4 images should align properly. If not, find out why and also fix it.
review-apple-bad-4set

**Please submit another code review request after fixing those issues**

Happy Coding :v: 
**Ebuka Umeokonkwo**
 [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Slack](https://app.slack.com/team/UFDKXE68L) | [Twitter](https://twitter.com/ebukaume)

**Review by Redvanisation — CHANGES_REQUESTED** [review]
# Changes required  🛑

### Hello @fdezcaminero   🌟

**Good work implementing the changes, but we need to fix the issues below before moving on ⬇️** 

1. The page content should be centered, on my screen it looks like what's on the image below:
Screen Shot 2019-11-06 at 7 51 10 PM

2. The third main text should be aligned, it's currently showing on the right side of the screen.

3. **[OPTIONAL]:** You also could use headings instead of images for the main text.

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  height: 20px;
+}
+
+.apple-search {
+  background: gray;
+  border-radius: 8px;
+  height: 25px;
+  left: 830px;
+  position: relative;
+  top: -22.5px;
```
Comment (Redvanisation) [inline css/styles.css]: The search input is not showing in the navbar, the reason for that is that you set the left positioning to `830px` and the top to `-22.5px`, one way to fix that is to remove lines 40 to 42.

**Review by Redvanisation — CHANGES_REQUESTED** [review]
# Changes required  🛑

### Hello @fdezcaminero   🌟

### Excellent work implementing the changes, however, there is still one issue that needs to be fixed below: ⬇️ 

1. The navbar should have a search input on the rightmost part, I see you have that on the HTML markup but it isn't showing up due to a CSS mistake, please check out my comment below for more on how to fix that.

2. Please remove the comment from the README file and keep only things related to the project features and description, any comments would be better made in the pull request.


### _After making the required changes please submit a new code review request_

### Good luck! 🍀

File: css/styles.css (hunk 1 above)
Comment (fdezcaminero) [reply css/styles.css]: I think I fixed it. I was using position relative. I erased those three lines of code as you said, and relied instead on the natural flow of HTML/CSS. Made the navbar bigger and eventually it aligned itself.

**Review by suretrust — APPROVED** [review]
Amazing job @fdezcaminero, :raised_hands: 

### Let's get this merged. :+1: 

_Good luck with your future projects and happy coding!_

**Pull request merged**